### DISCUSSION TITLE
Keyboard shortcuts not working on Ventura

### DISCUSSION THREAD

**Discussion by taschetto** [rxhanson/RectanglePro-Community#178] [Q&A] [answered]
Hi! I'm a new Mac user (coming from Linux). I bought Rectangle Pro. Keyboard shortcuts are doing nothing right now. Any thoughts?

![image](https://user-images.githubusercontent.com/5279182/203339388-d03cc7b2-f7a4-409c-ba70-f675b868a971.png)
![image](https://user-images.githubusercontent.com/5279182/203339449-3b67ba79-e997-47bc-9a83-40c8069904a1.png)

**Comment by rxhanson** [comment]
Thanks for purchasing. I’ll need a little more information to understand what is going on. 

If nothing is working, then it could be accessibility permissions quirks in macOS. You could try just restarting the app, or if it is not working on a specific app, try restarting that app. If that doesn’t work, try quitting the app and disabling then removing it from System Settings > Privacy & Security > Accessibility. Then relaunch the app and reenable in System Settings. 

If all else fails, view the logs: hold option and select view logging in the menu bar menu. Then reproduce the issue and post the logs here.

**Comment by taschetto** [comment, marked as answer]
Please disregard this, it was a Mac newbie error. I was mistaken between ⇧ and ⌃ while trying to invoke the shortcuts. I apologize for the inconvenience.

**Reply by taschetto** [reply]
Feel free to delete the discussion to reduce the noise.
